tv_apps/smart-system/fxa/js/screens/fxam_refresh_auth.js

```diff
@@ -84,13 +84,29 @@ var FxaModuleRefreshAuth = (function() {
       this.importElements(
         'fxa-pw-input-refresh',
         'fxa-show-pw-refresh',
-        'fxa-forgot-password-refresh'
+        'fxa-forgot-password-refresh',
+        'fxa-pw-clean-btn'
       );
 
       this.fxaPwInputRefresh.addEventListener('input', (function(event) {
```

Comment:
If it is an anonymous function, can we use arrow function here? I see there are different function style in this file. That would be great to make them consistent and avoid using ```bind```.